src/dependencies.jl

```diff
@@ -89,8 +89,12 @@ function get_latest_version!(deps, registries)
                 versions_toml_path = joinpath(
                     registry, packages[uuid]["path"], "Versions.toml"
                 )
-                versions = VersionNumber.(collect(keys(TOML.parsefile(versions_toml_path))))
-
+                versions_dict = TOML.parsefile(versions_toml_path)
+                exclude_yanked = true
```

Comment:
Make this an kwarg possibly? Is there ever a time you would not want this to be `true` though? I can't imagine.